docs/catalog/ai/leverage-sustainable-regions.md

```diff
@@ -0,0 +1,39 @@
+---
+version: 1.0
+submitted_by: navveenb
+published_date: tbd
+category: ai
+tags: 
+ - ai
+ - machine-learning
+ - role:data-scientist
+ - size:small
+---
+
+# Use sustainable regions for ML training 
+
+## Description
+Training an AI model has a significant carbon footprint. Leverage
+a region which offers carbon-free energy for running your ML training jobs.
+
+
+## Solution
+Use a cloud region which offers carbon-free energy for running your ML training workloads.
+
+
+## SCI Impact
+`SCI = (E * I) + M per R`
+
+For the SCI equation, using a sustainable region for ML training would impact the following:
+- 'E': Having a carbon free region for ML training, would reduce the carbon emissions of ML applications, decreasing the E number.
```

Comment:
I think this should be an `I` instead of `E`, right? The energy required across regions will be the same but the intensity will vary based on the emissions from the grid